product-catalog/app/com/commercetools/sunrise/productcatalog/productoverview/search/facetedsearch/categorytree/viewmodels/CategoryTreeFacetOptionViewModelFactory.java

```diff
@@ -25,15 +25,24 @@
     private final List<Locale> locales;
     private final CategoryTree categoryTree;
     private final ProductReverseRouter productReverseRouter;
+    private final Set<String> ignoredParams;
```

Comment:
Why not make it static better and declare it directly in here?
```java
private static final Set<String> IGNORED_PARAMS = Collections.singleton("page"); 
```
It saves some lines of code and it's a bit more efficient.